### DISCUSSION TITLE
MWCS window length

### DISCUSSION THREAD

**Discussion by Joel-Seism** [ROBelgium/MSNoise#279] [Q&A] [answered]
Hi I have a question related to MWCS window length determination.

When I read many papers, they determine MWCS window length as about 1, 3, other times of the longest period.

The reason was usually related to true velocity change and time resolution, but I could not find the way to consider them.

When I set mwcs_wlen, is there any way or method to determine which moving window length most suitable?

**Comment by ThomasLecocq** [comment, marked as answer]
a rule of thumb is that you want to have at least (if not 1.5x or more) times the longest period in that window. So for a classic 0.1-1.0Hz filter, I use to take a wlen=12 (1.2x the 10s wave).

**Reply by Joel-Seism** [reply]
I see. Then how about the maximum period? Is there any standard? Because some ppl use 3, 6 times while some ppl use 1.2 times as your examples. And how do you make the rule of thumb for checking whether the window length is suitable?

**Reply by ThomasLecocq** [reply]
as for window length: Yes, 3 or 6 periods are also good, because they could also prevent cycle skipping when using MWCS. I've never tested those. Of course, the wider the window, the smoother the delta_t measured. I personally have never tested all options, so I suspect it's worth exploring
about the total width of the coda window, mutliple wavelengths are e.g. recommended by 
Ugalde, A., Gaite, B., & Villaseñor, A. (2014). Temporal variations of seismic velocity at Paradox Valley, Colorado, using passive image interferometry. Bulletin of the Seismological Society of America, 104(3), 1088-1099

**Reply by Joel-Seism** [reply]
I got it. Thank you for letting me know :) I will read the paper and explore more.
